src/main/java/de/tum/in/www1/artemis/domain/Result.java

```diff
@@ -107,6 +110,12 @@
     @Column(name = "example_result")
     private Boolean exampleResult;
 
+    @OneToOne(mappedBy = "result", cascade = CascadeType.ALL, fetch = FetchType.LAZY)
```

Comment:
Why lazy? Should not be a huge overhead to have it eager imo as it is OneToOne :)